### PR TITLE
524 subject wrongly typed person

### LINKED ISSUE #524: Subject wrongly typed as "Person"
Reported by Mr. Greßhoff. In http://lobid.org/resources/HT007331917.json the subject "Droste-Hülshoff, Annette von: Die Judenbuche" is typed as Work and Person although it is only a Work.

### PR DESCRIPTION
**Pull request by ChristophEwertowski**
Resolves #524

### CODE REVIEW

**Comment by ChristophEwertowski** [comment]
Used a new forked branch from the master. So the commented, unused code is removed now.

**Comment by dr0i** [comment]
OK.Now I see only added data. There is nothing else changed.

**Comment by ChristophEwertowski** [comment]
Used a new branch because merging with the master branch would have been too tedious. On test [HT007331917 (test, json](http://test.lobid.org/resources/HT007331917.json) looks good. Reminder: Type `Person` from `Droste-Hülshoff, Annette von: Die Judenbuche`, which is a `Work` should be removed.

File: src/test/resources/output/json/00733/HT007331917.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      "label" : "Holzstich",
+      "source" : {
+        "id" : "http://d-nb.info/gnd/7749153-1",
+        "label" : "Gemeinsame Normdatei (GND)"
+      },
+      "type" : [ "SubjectHeading" ]
+    }, {
+      "dateOfBirth" : "1797",
+      "dateOfDeath" : "1848",
+      "id" : "http://d-nb.info/gnd/4113356-0",
```
Comment (dr0i) [inline src/test/resources/output/json/00733/HT007331917.json]: The work shouldn't have a birth/death date. But then, it's catalogued in that way. So. Be it.

**Comment by ChristophEwertowski** [comment]
There is already an issue for it: https://github.com/hbz/lobid-resources/issues/529

**Comment by dr0i** [comment]
Deployed to production, see  http://lobid.org/resources/HT007331917.json. 
Closed.

**Pull request merged**